### PR TITLE
[Enhancement] (nereids)implement showWaringErrorsCommand in nereids

### LINKED ISSUE #42752: [Enhancement] (nereids)implement showWaringErrorsCommand in nereids
showWaringErrors command is supported in legacy planner but not in nereids(new doris fe planner). you need implement the following showWaringErrors command in nereids:
unsupportedshowWaringErrorsStatement
``` SHOW (WARNINGS | ERRORS) limitClause?                                         ```

### PR DESCRIPTION
**Pull request by msridhar78**
### What problem does this PR solve?

Issue Number: close #42752 

Related PR: #xxx

Problem Summary:

[Enhancement] (nereids)implement showWaringErrorsCommand in nereids

### Release note

None

### Check List (For Author)

- Test 
    - [x] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by msridhar78** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowWarningErrorsCommand.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        super(PlanType.SHOW_WARNING_ERRORS_COMMAND);
+        this.isWarning = isWarning;
+        this.limit = limit;
+    }
+
+    @Override
+    public ShowResultSet doRun(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        List<List<String>> rowSet = Lists.newArrayList();
+        // Only success
+        return new ShowResultSet(this.getMetaData(), rowSet);
```
Comment (Vallishp) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowWarningErrorsCommand.java:57]: we can directly use META_DATA directly no need function.

Comment (msridhar78) [reply fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowWarningErrorsCommand.java]: done

**Comment by msridhar78** [comment]
run buildall

**Review by Vallishp — APPROVED** [review]
LGTM

**Review by starocean999 — APPROVED** [review]

**Pull request merged**